.github/workflows/ci.yml

```
@@ -102,3 +102,20 @@ jobs:
                   name: schematic-pdfs
                   path: schematic-pdfs
                   retention-days: 7
+
+    generate-traditional-boms:
+        name: Generate traditional BOM(s)
+        runs-on: ubuntu-20.04
+        steps:
+            - uses: actions/checkout@v2
+                with:
+                    submodules: recursive
```

It appears that GitHub actions is having trouble starting the CI workflow run, and interestingly the "Generate traditional BOM(s)" job isn't listed.

Last time this happened it was caused by a syntax error. The only thing I've noticed is an indentation inconsistency (that was present in the code suggestion in #20) that has been corrected in the suggestion below. I don't know if YAML is indentation sensitive so this may not be the cause of the problem.
```suggestion
            - uses: actions/checkout@v2
              with:
                  submodules: recursive
```
I will review the remainder of the PR once the CI workflow has been successfully run.